### PR TITLE
fix: transfer from micromatch to picomatch

### PR DESCRIPTION
**Pull request by Zxilly**
ref to discussion at https://github.com/casbin/node-casbin/issues/257#issuecomment-829001237

This will remove dependency on `node/util`

Signed-off-by: Zxilly <zhouxinyu1001@gmail.com>

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1104,13 +1099,6 @@
...
-"@types/micromatch@^4.0.1":
-  version "4.0.1"
-  resolved "https://registry.npm.taobao.org/@types/micromatch/download/@types/micromatch-4.0.1.tgz?cache=0&sync_timestamp=1596841637355&other_urls=https%3A%2F%2Fregistry.npm.taobao.org%2F%40types%2Fmicromatch%2Fdownload%2F%40types%2Fmicromatch-4.0.1.tgz#9381449dd659fc3823fd2a4190ceacc985083bc7"
-  integrity sha1-k4FEndZZ/Dgj/SpBkM6syYUIO8c=
-  dependencies:
-    "@types/braces" "*"
-
 "@types/minimatch@^3.0.3":
   version "3.0.3"
   resolved "https://registry.npm.taobao.org/@types/minimatch/download/@types/minimatch-3.0.3.tgz?cache=0&sync_timestamp=1596839015387&other_urls=https%3A%2F%2Fregistry.npm.taobao.org%2F%40types%2Fminimatch%2Fdownload%2F%40types%2Fminimatch-3.0.3.tgz#3dca0e3f33b200fc7d1139c0cd96c1268cadfd9d"
```
Comment (hsluoyz) [inline yarn.lock:1104]: @Zxilly plz make a new PR to update all registries from `registry.npm.taobao.org` to `registry.npmjs.org`

**Comment by hsluoyz** [comment]
@Zxilly plz resolve conflicts:

image

**Comment by Zxilly** [comment]
@hsluoyz resolved

**Review by hsluoyz — CHANGES_REQUESTED** [review]
yarn.lock is not updated

**Comment by nodece** [comment]
@Zxilly use `fix` instead `refactor`.

**Review by nodece — APPROVED** [review]

**Comment by Zxilly** [comment]
@hsluoyz the new package is recursively import in the past. The removed package is also in the dependency of semantic. So there's no need to update yarn.lock.

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**